### PR TITLE
Doc-tweaks

### PR DESCRIPTION
**Pull request by tzaffi**
## Summary

Doc updates.

This is a draft but I welcome comments.

I'm adding several `# TODO`'s as I go along to remind me to figure out the answers.

## Test Plan

No changes in code.

### CODE REVIEW

File: docs/Configuration.md [hunk 1]
```diff
@@ -32,8 +32,11 @@ metrics:
...
 # Define one importer.
 importer:
-    name:
+    name: "name for the importer"
     config:
+      mode: "archival or follower"
+      netaddr: "importer URL"
+      token: "importer API token"
 
 # Define one or more processors.
```
Comment (tzaffi) [inline docs/Configuration.md:41]: TODO: figure out what the processors and exporters look like and modify accordinging.

File: docs/plugins/algod.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
 # Algod Importer
 
-Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod/v2/). The node must be configured as an archival node in order to
-provide old blocks.
+Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod/v2/). The node must be configured as an archival node in order to provide old blocks.
+
+`TODO: we can't use a follower ?`
```
Comment (tzaffi) [inline docs/plugins/algod.md:5]: ^^^

File: docs/plugins/filter_processor.md [hunk 3]
```diff
@@ -2,10 +2,14 @@
 
 This is used to filter transactions to include only the ones that you want. This may be useful for some deployments
 which only require specific applications or accounts. Using the default configuration, results for a matched transaction
-would also include its transaction group and search is disable for inner transactions. 
+would also include its transaction group and search is disabled for inner transactions.
+
+`TODO: How to apply the following sections?`
```
Comment (tzaffi) [inline docs/plugins/filter_processor.md:7]: ^^^

File: docs/plugins/algod.md [hunk 4]
```diff
@@ -1,16 +1,17 @@
 # Algod Importer
 
-Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod/v2/). The node must be configured as an archival node in order to
-provide old blocks.
+Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod/v2/). The node must be configured as an archival node in order to provide old blocks.
```
Comment (algochoi) [inline docs/plugins/algod.md:3]: Not directly related to this PR but a suggested doc change: the algod links were changed very recently to drop `v2` in the docs:

```suggestion
Fetch blocks one by one from the [algod REST API](https://developer.algorand.org/docs/rest-apis/algod). The node must be configured as an archival node in order to provide old blocks.
```

File: docs/Configuration.md [hunk 5]
```diff
@@ -32,8 +32,11 @@ metrics:
 
 # Define one importer.
 importer:
-    name:
+    name: "name for the importer"
     config:
+      mode: "archival or follower"
+      netaddr: "importer URL"
+      token: "importer API token"
```
Comment (winder) [inline docs/Configuration.md:39]: mode/netaddr/token are specific to the algod importer

File: docs/plugins/algod.md (hunk 2 above)
Comment (winder) [reply docs/plugins/algod.md:5]: This is out of date. The plugin docs need some work: https://github.com/algorand/conduit/issues/28

File: docs/plugins/filter_processor.md (hunk 3 above)
Comment (winder) [reply docs/plugins/filter_processor.md:7]: We have a document for the filter processor now, and it seems like there's a lot of overlap: https://github.com/algorand/conduit/blob/master/docs/tutorials/FilterDeepDive.md

**Comment by tzaffi** [comment]
subsumed by #64

**Pull request closed without merging**